packages/ses/src/reporting.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @endo/no-polymorphic-call */
```

Comment:
Can you add a comment why this lint rule is disabled for the whole file instead of just individual `// eslint-disable-next-line .. directives?

Reply:
Better, I moved it to disable a block around `consoleReporter`.

Done